PSKoans/Koans/Cmdlets 2/AboutOutCmdlets.Koans.ps1

```diff
@@ -115,23 +115,20 @@ Context 'Out-* Cmdlets' {
             Out-String is a fantastic and simple way to turn complex objects into strings that look just
             like PowerShell's default console formatting.
         #>
-
+    
         It 'creates string representations of data' {
-            $HomeFolder = Get-Item -Path $HOME
-
-            $HomeFolder | Should -Not -BeNullOrEmpty
-            $String = $HomeFolder | Out-String
-
-            # Fill in this here-string with the data you'd see in your console from calling Get-Item $HOME
+            #Create a hashtable that pipes into Out-String
+            $String = '__' | Out-String
```

Comment:
If we're asking for a hashtable we should probably provide at minimum an empty hashtable or just an unquoted blank I guess as by this point I guess it's reasonable to expect them to know how to do that at this point.

Reply:
Agreed.